src/options.c

```diff
@@ -263,7 +263,8 @@ cmd_help (void)
   "                                    ranges. e.g., 192.168.0.1-192.168.0.10\n"
   "  -H --http-protocol=<yes|no>     - Set/unset HTTP request protocol if found.\n"
   "  -M --http-method=<yes|no>       - Set/unset HTTP request method if found.\n"
-  "  -o --output=file.html|json|csv  - Output either an HTML, JSON or a CSV file.\n"
+  "  -o --output=<filename|format>   - Output to stdout or the specificed file.\n"
+  "                                    Currently support HTML, JSON and CSV format.\n"
```

Comment:
Update:
Now I know `-o file.csv` will save the output to a file, while `-o csv` will directly display it in the terminal.
It's currently not included in the help message. 😅

I need your assistance in improving the proposed wording, thanks. 🙇‍♂️

Reply:
I believe we can include a third line to show both options, and a brief example, like the one you posted above, would suffice. Thanks!